Casks/mellel.rb

```diff
@@ -0,0 +1,32 @@
+cask "mellel" do
+  version "51012"
```

Comment:
IF we're going by full version it should be
```suggestion
  version "5.1.0.51012"
```

Reply:
Ok, I've adapted the cask to use the full version string, as you suggested.